abilitybot/startScenarioExample.js

```diff
@@ -1,12 +1,19 @@
 import { CharInfo } from "../bossroom";
 
 export function configureBot(rg) {
-    rg.characterType = CharInfo.type[0]; // healer
+    rg.characterConfig = {
+        characterType: CharInfo.type[0]
+    }; // healer
 }
 
 export async function startScenario(rg) {
 
-    const charType = CharInfo.type.indexOf(rg.characterType);
+    let charType = CharInfo.type.indexOf(rg.characterConfig.characterType);
+    const overrideType = JSON.parse(rg.characterConfig).characterType;
+    if (overrideType) {
+        charType = CharInfo.type.indexOf(overrideType);
```

Comment:
Shouldn't all of this just be
```
const characterType = CharInfo.type.indexOf(rg.characterConfig.characterType);
```

you don't need to parse the JSON because it should already be an object on `rg`

Reply:
Good catch! Removed since this was just a duplicate anyways!